Language/Structure/Control Structure/switchCase.adoc

```diff
@@ -52,7 +52,6 @@ switch (var) {
 === Returns
 Nothing
 
---
```

Comment:
This [block](https://docs.asciidoctor.org/asciidoc/latest/blocks/) delimiter markup is required in order for the document to be valid AsciiDoc.